TestVectors/Ed25519Signature2020/canonDocEdSig.txt

```diff
@@ -0,0 +1,8 @@
+<did:example:abcdefgh> <https://schema.org/alumniOf> "The School of Examples" .
+<urn:uuid:58172aac-d8ba-11ed-83dd-0b3aef56cc33> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <https://www.w3.org/2018/credentials#VerifiableCredential> .
+<urn:uuid:58172aac-d8ba-11ed-83dd-0b3aef56cc33> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <https://www.w3.org/ns/credentials/examples#AlumniCredential> .
+<urn:uuid:58172aac-d8ba-11ed-83dd-0b3aef56cc33> <https://schema.org/description> "An minimum viable example of an Alumni Credential." .
```

Comment:
```suggestion
<urn:uuid:58172aac-d8ba-11ed-83dd-0b3aef56cc33> <https://schema.org/description> "A minimum viable example of an Alumni Credential." .
```

Reply:
^ probably not a good idea to change the n-quads, this change should be made to the JSON-LD.

Reply:
OK, so this is an artifact, not a managed document, and should probably be added to `.gitignore`.

Reply:
No, it is a referenced (included document), so cannot be added to .gitignore. When we update the base test vector document it will be re-created but that does not happen in this repo.